lib/Frenetic_Fdd.ml

```diff
@@ -65,34 +66,24 @@ module Field = struct
     | Location -> "Location"
     | VFabric -> "VFabric"
 
```

Comment:
Adding or removing fields just got a lot easier! There are no more nasty dependencies between the definitions of `type t`, `num_fields`, and `all_fields`.